qplist.pri

```diff
@@ -0,0 +1 @@
+LIBS += 
```

Comment:
By definition, a pod contains a pro for building the pod and a pri that will be included in the targets pro. If you have nothing to propagate, you can leave it empty also (which expresses there is nothing to do). So one could remove that line entirely and leave an empty pri.